cypress/support/commands.js

```diff
@@ -79,7 +80,7 @@ Cypress.Commands.overwrite("request", (originalFn, ...args) => {
 });
 
 Cypress.Commands.add("deleteAllIndices", () => {
-  cy.request("DELETE", `${Cypress.env("opensearch")}/index*,sample*,opensearch_dashboards*`);
+  cy.request("DELETE", `${Cypress.env("opensearch")}/index*,sample*,opensearch_dashboards*,test*`);
```

Comment:
**Q:** What does this do?

Reply:
there was an existing 'deleteAllIndices' command added already, I just added my own for testing purposes.  It deletes all indices within a test with specified prefixes.